### PR TITLE
improvement: Add scopes for debugging and sanity

### PR DESCRIPTION
**Pull request by OAuthBringer**
Hard to debug in staging without being able to quickly see things.

### CODE REVIEW

File: app/models/channel.rb [hunk 1]
```diff
@@ -112,6 +113,17 @@ class Channel < ApplicationRecord
     where.not(contested_by_channel_id: nil).where("contest_timesout_at < ?", Time.now)
   }
 
+  scope :missing_deposit_id, -> { where(deposit_id: nil) }
+  scope :using_active_bitflyer_connection, -> {
+    joins(:publisher)
```
Comment (OAuthBringer) [inline app/models/channel.rb:118]: this might actually be redundant but the specs work.  I could remove it later

Comment (tsmartt) [reply app/models/channel.rb:118]: This would be useful to have for Uphold and Gemini too, eventually.

Comment (OAuthBringer) [reply app/models/channel.rb:118]: @tsmartt For sure, pattern is the same.

Comment (tsmartt) [reply app/models/channel.rb:118]: maybe just `with_active_bitflyer_connections`

Comment (OAuthBringer) [reply app/models/channel.rb:118]: Yeah, some standard would be better.  Not overthinking it atm.  If/when we add the others we can clean it up.

**Review by jlbyrne — APPROVED** [review]
LGTM!

**Pull request merged**